resources/views/pages/rock-the-vote-reports/create.blade.php

```diff
@@ -35,6 +32,8 @@
         <div>
             <input type="submit" class="btn btn-primary btn-lg" value="Create">
         </div>
+        <hr />
+        @include('pages.partials.rock-the-vote.create', ['config' => $config])
```

Comment:
I don't see where `config` is used in `partials/rock-the-vote/create.blade.php`?

Reply:
It's used to display the various source, action, configuration settings:
https://github.com/DoSomething/chompy/blob/578344bb61de1c34c47802bf468e4ec7ba742652/resources/views/pages/partials/rock-the-vote/create.blade.php#L6

Reply:
oh my bad! I thought it was new so I was looking for it in the updated pieces. Thanks!